src/pages/home/categories/categories.js

```diff
@@ -2,24 +2,27 @@ import React from 'react';
 import { useSelector } from 'react-redux';
 import Typography from '@material-ui/core/Typography';
 
-import LoadingBar from '../../../components/LoadingBar';
-import CategoryItem from '../category-item';
+import LoadingBar from '../../../components/loading-bar';
+import CategoryItem from './category-item';
 import { useStyles } from './categories.style';
-import { LANGUAGE, HOMEPAGE_TITLES } from '../../../configs';
+import { HOMEPAGE_TITLES } from '../../../configs';
 
 const Categories = () => {
```

Comment:
Please, rename component and its files to `CategoriesList` to reduce duplication of code with a name of the state key